plugins/shorten_urls.lua

```diff
@@ -74,8 +74,11 @@ return {
 					end
 				end
 				if #msg > 0 then
-					msg = sender[1] .. ": " .. msg
-					irc:PRIVMSG(origin, msg)
+					-- This string contains an zero-width space character
+					-- which might be invisible in your editor
+					local nick = sender[1]:gsub("(.)(.+)", "%1​%2")
```

Comment:
If the username is a single character then this will return `nil` (and hence fail in the following concatenation).

Use `*` instead of `+` in the gsub instead. also considering anchoring the gsub.